test/docker-compose.yml

```diff
@@ -18,4 +18,8 @@ services:
     - clamav-mirror
     restart: unless-stopped
   clamav-mirror:
```

Comment:
for info: this allows you to save the clamav database while testing locally so you don't get throttled